app/src/test/java/com/ikarabulut/BoardTest.java

```diff
@@ -69,4 +69,121 @@ void setRandomGrid() {
         assertFalse(isDeadBoard);
     }
 
-}
\ No newline at end of file
+    @Test
+    @DisplayName("A cell that is dead can be set to alive")
```

Comment:
```suggestion
    @DisplayName("The board can bring a cell to life at a given location")
```

Just trying to differentiate between the board being able to tell a cell to come alive and the feature that will be in the Cell class, where it can bring itself to life.

Reply:
I like this naming better. Will include this change in my next merge